proxy/hdrs/CMakeLists.txt

```diff
@@ -46,3 +46,26 @@ target_link_libraries(hdrs
     PRIVATE
         ts::inkevent
 )
+
+add_executable(test_proxy_hdrs
+    unit_tests/test_HdrHeap.cc
+    unit_tests/test_Hdrs.cc
+    unit_tests/test_HdrUtils.cc
+    unit_tests/test_HdrHeap.cc
+    unit_tests/test_HeaderValidator.cc
+    unit_tests/test_Huffmancode.cc
+    unit_tests/test_mime.cc
+    unit_tests/test_URL.cc
+    unit_tests/unit_test_main.cc
+)
+target_link_libraries(test_proxy_hdrs PRIVATE ts::hdrs ts::tscore ts::inkevent catch::catch)
+add_test(NAME test_proxy_hdrs COMMAND $<TARGET_FILE:test_proxy_hdrs>)
```

Comment:
Is the generator expression necessary? My understanding of the `add_test` docs was that if the command is the name of a target, it automatically uses that targets binary output.

Reply:
I don't think it is necessary but I was just following suit from other invocations of `add_test`